rectools/models/nn/transformers/base.py

```diff
@@ -596,20 +606,28 @@ def _model_from_checkpoint(cls, checkpoint: tp.Dict[str, tp.Any]) -> tpe.Self:
             item_external_ids=item_external_ids,
             model_config=model_config,
         )
+
+        # save checkpoint to temp file to be able to use it in trainer
+        with NamedTemporaryFile() as f:
```

Comment:
Let's accept an optional ckpt_path argument to `_model_from_checkpoint` and pass it from `load_from_checkpoint` so that we don't save another chekpoint when we have user already passing it.